### PR TITLE
Explain how types can be used for storage enhancements

### PR DESCRIPTION
**Pull request by KnorpelSenf**
Explains how to use TS with storage enhancements

### CODE REVIEW

**Review by roj1512 — APPROVED** [review]

**Comment by KnorpelSenf** [comment]
@niusia-ua please fix the formatting

File: site/docs/uk/plugins/session.md [hunk 1]
```diff
@@ -581,20 +587,22 @@ bot.start();
...
-    initial: () => ({ prop: 0 }),
-    storage: freeStorage(bot.token),
-  },
-  baz: {},
-}));
+bot.use(
+  session({
+    type: "multi",
+    foo: {
+      // це також типове значення значення
```
Comment (LWJerri) [inline site/docs/uk/plugins/session.md:594]:
```suggestion
      // це також типове значення
```

I think it's still clear what we're talking about if we remove one word.

File: site/docs/uk/plugins/session.md [hunk 2]
```diff
@@ -833,6 +847,45 @@ const enhanced = enhanceStorage({
...
+      return { count: 0 };
+    },
+    storage: enhanceStorage({
+      storage: new MemorySessionStorage<Enhance<SessionData>>(),
+      millisecondsToLive: 60_000,
+    }),
+  }),
+);
+
+bot.on("message", (ctx) => ctx.reply(`Кількість чатів ${ctx.session.count++}`));
```
Comment (LWJerri) [inline site/docs/uk/plugins/session.md:880]:
```suggestion
bot.on("message", (ctx) => ctx.reply(`Кількість повідомлень ${ctx.session.count++}`));
```

**Review by LWJerri — CHANGES_REQUESTED** [review]
Resolve all my comments, please and I approve this PQ.

File: site/docs/uk/plugins/session.md (hunk 1 above)
Comment (niusia-ua) [reply site/docs/uk/plugins/session.md:594]: I corrected a language mistake and did not notice the duplication of the word. There is no hidden meaning here 😅

File: site/docs/uk/plugins/session.md [hunk 3]
```diff
@@ -833,6 +847,45 @@ const enhanced = enhanceStorage({
...
+      return { count: 0 };
+    },
+    storage: enhanceStorage({
+      storage: new MemorySessionStorage<Enhance<SessionData>>(),
+      millisecondsToLive: 60_000,
+    }),
+  }),
+);
+
+bot.on("message", (ctx) => ctx.reply(`Кількість повідомлень в чаті ${ctx.session.count++}`));
```
Comment (LWJerri) [inline site/docs/uk/plugins/session.md:880]:
```suggestion
bot.on("message", (ctx) => ctx.reply(`Кількість чатів ${ctx.session.count++}`));
```

My bad, see: https://t.me/grammyjs/225933 -> https://t.me/grammyjs/226006

File: site/docs/uk/plugins/session.md (hunk 2 above)
Comment (LWJerri) [reply site/docs/uk/plugins/session.md:880]: https://github.com/grammyjs/website/pull/1019/commits/c054f27c04a75299f6cf92fed5ba61b2c7fbae7d#r1536804513

**Review by LWJerri — APPROVED** [review]
:shipit:

**Comment by KnorpelSenf** [comment]
Please add your language label when you're done :)

**Review by dzikrl — DISMISSED** [review]

File: site/docs/id/plugins/session.md [hunk 4]
```diff
@@ -839,6 +853,45 @@ const enhanced = enhanceStorage({
...
 Tidak peduli seberapa besar data session sebuah chat telah berubah, segera setelah ia dibaca, ia akan dimigrasikan ke versi yang telah ditentukan hingga ia menggunakan struktur yang terbaru.
 
+### Type untuk Peningkatan Storage
+
+Ketika kamu menggunakan peningkatan storage, storage adapter akan menyimpan lebih banyak data daripada data session.
+Misalnya, suatu data disimpan bersama dengan waktu kapan session terakhirnya, sehingga nanti ia akan [kadaluarwasa](#timeout) dengan benar setelah waktunya habis.
+Dalam beberapa kasus, TypeScript akan menebak type yang sesuai untuk storage adapter-mu.
+Namun, sering kali kita yang membuat type annotation untuk data session-nya secara eksplisit pada beberapa tempat.
+
+Contoh kode dibawah ini mengilustrasikan bagaimana cara menggunakan peningkatan timeout dengan type yang benar pada TypeScript.
```
Comment (KnorpelSenf) [inline site/docs/id/plugins/session.md:863]:
```suggestion
Contoh kode dibawah ini mengilustrasikan bagaimana cara menggunakan peningkatan timeout dengan type yang benar pada TypeScript.
```

**Review by dzikrl — APPROVED** [review]

**Pull request merged**